packages/react/src/PricingOptions/PricingOptions.module.css

```diff
@@ -0,0 +1,339 @@
+/* ---------------------------------------------------------- */
+/* Pricing options component                                  */
+/* ---------------------------------------------------------- */
+/* Table of contents:                                         */
+/* 1. Container                                               */
+/* 2. Layout: default                                         */
+/* 3. Layout: cards                                           */
+/* 4. Single item                                             */
+/* 5. Multiple items                                          */
+/* 6. Item inner layout                                       */
+/* 7. Content                                                 */
+/* 8. Feature list                                             */
```

Comment:
I do love how organized this is, but I've noticed it's hard to maintain as small changes get made to this file. Using Prose as reference point. Could we drop the ToC + ordered numbers but keep the section headings instead?